packages/wdio-sync/src/index.js

```diff
@@ -71,7 +71,9 @@ const executeAsync = async function (fn, repeatTest = 0, args = []) {
             return await executeAsync(fn, --repeatTest, args)
         }
 
-        e.stack = e.stack.split('\n').filter(STACKTRACE_FILTER_FN).join('\n')
+        if (e.stack) {
+           e.stack = e.stack.split('\n').filter(STACKTRACE_FILTER_FN).join('\n')
```

Comment:
it should be 12 spaces instead of 11 here
```
/home/travis/build/webdriverio/webdriverio/packages/wdio-sync/src/index.js
  75:1  error  Expected indentation of 12 spaces but found 11  indent
```